repository/copier.go

```diff
@@ -131,7 +137,7 @@ func (c *HDFSCopier) CopyToRemote(src, dst string, localFs billy.Filesystem) (er
 		return err
 	}
 
-	pCopy := p + ".copy"
+	pCopy := path.Join(c.tmpBase, bDst+".copy")
```

Comment:
I'm not sure buckets are needed in the temporary directory:

```
pCopy := path.Join(c.tmpBase, dst+".copy")
```

otherwise LGTM :+1:

Reply:
Yup, you're totally right.